ruby/ql/test/library-tests/dataflow/barrier-guards/barrier-guards.rb

```diff
@@ -102,3 +102,9 @@
 else
     foo
 end
+
+if x or y then
+    foo
+else
+    foo
```

Comment:
Let's use a different name in the else clause to make it easier to validate the expected output.

```suggestion
    bar
```